src/main/java/kr/sesac/aoao/server/item/service/ItemServiceImpl.java

```diff
@@ -0,0 +1,78 @@
+package kr.sesac.aoao.server.item.service;
+
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import kr.sesac.aoao.server.dino.exception.DinoErrorCode;
+import kr.sesac.aoao.server.dino.repository.DinoEntity;
+import kr.sesac.aoao.server.dino.repository.DinoJpaRepository;
+import kr.sesac.aoao.server.global.exception.ApplicationException;
+import kr.sesac.aoao.server.item.controller.dto.GetItemInfoResponse;
+import kr.sesac.aoao.server.item.controller.dto.UseItemNumResponse;
+import kr.sesac.aoao.server.item.exception.ItemErrorCode;
+import kr.sesac.aoao.server.item.repository.ItemEntity;
+import kr.sesac.aoao.server.item.repository.ItemJpaRepository;
+import kr.sesac.aoao.server.item.repository.UserItemEntity;
+import kr.sesac.aoao.server.item.repository.UserItemJpaRepository;
+import kr.sesac.aoao.server.user.exception.UserErrorCode;
+import kr.sesac.aoao.server.user.repository.UserEntity;
+import kr.sesac.aoao.server.user.repository.UserJpaRepository;
+import lombok.RequiredArgsConstructor;
+
+@Service
+@Transactional
+@RequiredArgsConstructor
+public class ItemServiceImpl implements ItemService {
+	private final ItemJpaRepository itemRepository;
+	private final UserJpaRepository userRepository;
+	private final UserItemJpaRepository userItemRepository;
+	private final DinoJpaRepository dinoRepository;
+
+	/**
+	 * 아이템 정보 조회
+	 * @since 2024.01.22
+	 * @return GetItemInfoResponse
+	 * @author 김은서
+	 */
+	@Override
+	public GetItemInfoResponse getItemInfo(Long id) {
+		ItemEntity item = itemRepository.findById(id)
+			.orElseThrow(()-> new ApplicationException(ItemErrorCode.NOT_FOUND_ITEM));
+		return new GetItemInfoResponse(
+			item.getId(),
+			item.getName(),
+			item.getPrice(),
+			item.getExp()
+		);
+	}
```

Comment:
```suggestion
		return new GetItemInfoResponse(item);
	}
```
이렇게 객체를 보낸다음에 
```
@Getter
@RequiredArgsConstructor
public class GetItemInfoResponse {
	private final Long id;
	private final String name;
	private final int price;
	private final int exp;
	public GetItemInfoResponse(ItemEntity item){
		this.id = item.getId();
		this.name = item. getName();
		this.price = item. getPrice();
		this.exp = item. getExp();
	}
}
```

이런식으로 하면 코드가 더 깔끔할 것 같습니당

Reply:
> 해당 부분은 좀 더 공부하고 적용 하겠습니다.😱